lib/philomena_web/views/admin/artist_link_view.ex

```diff
@@ -21,10 +21,17 @@ defmodule PhilomenaWeb.Admin.ArtistLinkView do
     |> String.capitalize()
   end
 
-  def link_scope(conn) do
-    case conn.params["all"] do
-      nil -> []
-      _val -> [all: true]
+  def scope(conn) do
+    []
+    |> scope(conn, "lq", :lq)
+    |> scope(conn, "all", :all)
+  end
+
+  defp scope(list, conn, key, key_atom) do
+    case conn.params[key] do
+      nil -> list
+      "" -> list
+      val -> [{key_atom, val} | list]
```

Comment:
Technically not the same behavior as previously. Your code sets the value to `conn.params[key]`, the old code set it to `true`.

Reply:
It doesn't matter. The controller only checks for presence, not "true"